client/views/select2.jade

```diff
@@ -0,0 +1,6 @@
+template(name="select2")
+  select.form-control.select2(
+    id=selectId
+    name=name
+    multiple=multiple
+    required=required)
```

Comment:
We should probably show a placeholder? I think in order for it to show, an empty `option` element has to be given?